change_log/next/FE-2177-SimpleColorPicker-interface-refactor.yml

```diff
@@ -0,0 +1 @@
+Improvement: "Changed SimpleColorPicker so that it resembles current RadioButtonGroup interface. (Component: SimpleColorPicker)"
```

Comment:
Improvement is not one of the default headings, please could you pick from
  - Breaking Changes
  - New Features
  - New Components
  - Deprecated Components
  - Styling Changes
  - Bug Fixes
  - Dependency Updates
  - Demo Site Updates
  - Experimental
  - Other
  - Removed

Ditto for the other change logs.